ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env

```diff
@@ -1 +1,4 @@
-discussion_forum_token= {{ discussion_forum_token }}
\ No newline at end of file
+sunbird_discussion_forum={{ discussionsmw_sunbird_discussion_forum }}
+authorization_token={{ discussionsmw_nodebb_authorization_token }}
+mongodb_connection_url={{ discussionsmw_mongodb_connection_url }} # the connection url should be 'mongodb://host:port'
+discussions_middleware={{ discussionsmw_service_url }}  # this is portal env
```

Comment:
why do we need a loop back url ?

Reply:
this is env variable which we are using in portal 
from the portal when we call discussion api it will go to the portal from the portal backend and from there we are calling the discussion middleware service and this service url we added in portal env file

Reply:
if this is a portal env then it should be added to player.env file @VenkatKadiveti